apps/omg_api/lib/state/core.ex

```diff
@@ -395,26 +395,28 @@ defmodule OMG.API.State.Core do
   bare `EthereumEventListener` or `ExitProcessor`
   """
   @spec exit_utxos(exiting_utxos :: [Utxo.Position.t()] | [exit_t()], state :: t()) ::
-          {:ok, {[exit_event], [db_update]}, new_state :: t()}
+          {:ok, {[exit_event], [db_update], {list(Utxo.Position.t()), list(Utxo.Position.t())}}, new_state :: t()}
   def exit_utxos([%{utxo_pos: _} | _] = exit_infos, %Core{} = state) do
     exit_infos
     |> Enum.map(&Utxo.Position.decode(&1.utxo_pos))
     |> exit_utxos(state)
   end
 
   def exit_utxos(exiting_utxos, %Core{utxos: utxos} = state) do
-    exiting_utxos = Enum.filter(exiting_utxos, &utxo_exists?(&1, state))
+    _ = if exiting_utxos != [], do: Logger.info(fn -> "Recognized exits #{inspect(exiting_utxos)}" end)
```

Comment:
if statement should be inside function

Reply:
no, it should be outside, otherwise we'd get an empty log line